data/api/application-service/definitions/location.yaml

```diff
@@ -15,18 +15,21 @@ properties:
   alias:
     description: An alias for a matrix room.
     type: string
-    example: "#freenode_#matrix:matrix.org"
+    examples:
+      - "#freenode_#matrix:matrix.org"
```

Comment:
Switching from `example` to `examples` in schemas was not strictly necessary, because the field is only deprecated in `OpenAPI 3.1`, but I thought it made sense to already switch to the recommended format.

Reply:
Not a hard objection, but anything we can do to reduce the size of this diff to make it easier to review would be a good thing. If it's not a huge amount of work, I would defer this for a future PR.